timestamp-api/src/main/java/com/palantir/timestamp/TimestampBoundStoreEntry.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.timestamp;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class TimestampBoundStoreEntry {
+    private static final Long INITIAL_VALUE = 10000L;
```

Comment:
Hmm, I'd be hesitant about defining this to hold across timestamp bound stores (it is not the case for `PaxosTimestampBoundStore` and `InMemoryTimestampBoundStore`). It would seem to me that the initial value would be something specific to an implementation?